packages/rango-sdk/src/services/client.ts

```diff
@@ -121,6 +125,30 @@ export class RangoClient {
     return axiosResponse.data
   }
 
+  public async getAllRoutes(
+    requestBody: MultiRouteRequest,
+    options?: RequestOptions
+  ): Promise<MultiRouteResponse> {
+    const axiosResponse = await this.httpService.post<MultiRouteResponse>(
+      `/routing/bests?apiKey=${this.apiKey}`,
+      requestBody,
+      { headers: { 'X-Rango-Id': this.deviceId }, ...options }
+    )
+    return axiosResponse.data
+  }
+
+  public async confirmMultiRouteRequest(
```

Comment:
As you've mentioned earlier, if it is possible to call this API for non-multi route requests as well, we could change it to `confirmRouteRequest`.